web/src/app/admin/users/users-list.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from "@angular/router";
+import { UserViewModel } from '../../webapi/models/user-view-model';
+import { UserServiceApp } from './shared/user.serviceApp';
+
+@Component({
+    selector: 'users-list',
+    templateUrl: 'users-list.component.html'
+})
+
+export class UsersListComponent implements OnInit {
+
+    users: UserViewModel[] = [] as UserViewModel[];
+
+    constructor(private userService: UserServiceApp, private router: Router) {
```

Comment:
service variable name should be same as service name

Reply:
Done